provider-ci/internal/pkg/templates/base/.github/workflows/build_provider.yml

```
@@ -58,6 +58,27 @@ jobs:
         uses: ./.github/actions/setup-tools
         with:
           tools: pulumictl, go
+          cache-go: false
```

```suggestion
          cache-go: false # use per-platform/arch caches instead
```

just a quick comment to unconfuse a casual reader

---

Yep